git-hooks/pre-commit.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
I don't think there is a fool proof solution but we should try to make it hard for you to commit secrets.